common/src/main/java/com/microsoft/identity/common/internal/ui/webview/AzureActiveDirectoryWebViewClient.java

```diff
@@ -631,6 +669,21 @@ public void run() {
         }, TimeUnit.SECONDS.toMillis(THREAD_SLEEP_FOR_INTENT_LAUNCH_MS));
     }
 
+    // Opens the Google enrollment URL in the browser or the default intent handler (like DPC)
+    private void openGoogleEnrollmentUrl(@NonNull final String url) {
+        final String methodTag = TAG + ":openGoogleEnrollmentUrl";
+        Logger.info(methodTag, "Opening Google enrollment URL");
+        try {
+            final Intent intent = new Intent(Intent.ACTION_VIEW, Uri.parse(url));
+            // Add flags to ensure the activity is launched in a new task and clears the current task stack.
+            // Important for the enrollment flow to work correctly.
+            intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TASK);
+            getActivity().startActivity(intent);
+        } catch (android.content.ActivityNotFoundException e) {
+            Logger.error(methodTag,"Failed to open the intent for google enrollment.", e);
```

Comment:
In this case are we still returning the MDM_FLOW started error back to the client app?

Reply:
Yes this is still same as what I introduced in initial webcp support PR. Only difference is adding some intent flags. The calling method returns MDM_FLOW error